lms/services/vitalsource/_client.py

```diff
@@ -161,21 +160,30 @@ def get_user_credentials(self, user_reference: str) -> dict:
 
         raise VitalSourceError(error_code="vitalsource_user_not_found")
 
-    @staticmethod
-    def _is_resource_not_found(err: ExternalRequestError) -> bool:
-        """Get if an exception is due to a missing resource or a bad URL."""
+    @classmethod
+    def _handle_book_errors(cls, book_id: str, err: ExternalRequestError):
+        if json_errors := cls._get_json_errors(err):
+            if "Catalog not found" in json_errors:
```

Comment:
Do we know what this means? What's the solution? Needs VS intervention?

We could expand with a comment here if that's the case.

Reply:
Eh... not really. Apparently the "catalog was not intialised" the solution is to "initialise the catalog". This isn't something which really means much to us. I'll try and add a bit more info.